### PR TITLE
Add onboarding information for tenancy and leaseholds team

### PR DESCRIPTION
**Pull request by bogdan-zaharia**
## Describe this PR

Linked to this ticket: https://hackney.atlassian.net/browse/DOC-564

### *What changes have we introduced*

Added onboarding information for Tenancy and Leaseholds team. 

> Finished adding the config!)

Please check with Lewis the team name before merging. 

> They would like to be called Leasehold and Right to Buy

Also, this PR should be merged at the same time as the PR containing the onboarding information in the frontend.

> PR for the frontend exists -- being reviewed.

### CODE REVIEW

**Review by andrewclowes — APPROVED** [review]

File: EvidenceApi/StaffSelectedDocumentTypes.json [hunk 1]
```diff
@@ -435,5 +435,81 @@
...
+        "title": "Proof of previous Right to Buy discount",
+        "description": "If they have previously purchased a property under Right to Buy and completed Part D of the RTB1, resident to supply any relevant documentation. (website link to form)"
+      },
+      {
+        "id": "opt-out-rtb",
+        "title": "Joint tenant opting out of the Right to Buy",
+        "description": "Upload of completed document (website link to download)"
+      },
+      {
+        "id": "proof-of-marriage",
```
Comment (andrewclowes) [inline EvidenceApi/StaffSelectedDocumentTypes.json:489]: This should be `marriage-certificate`.

File: EvidenceApi/StaffSelectedDocumentTypes.json [hunk 2]
```diff
@@ -435,5 +435,81 @@
...
+        "id": "council-tax-bill",
+        "title": "Council tax bill",
+        "description": "Current financial year"
+      },
+      {
+        "id": "government-letter",
+        "title": "Government letter",
+        "description": "For example, benefit status letters"
+      }
+    ]
```
Comment (andrewclowes) [inline EvidenceApi/StaffSelectedDocumentTypes.json:523]: `permission-booklet` and `anti-money laundering form` are missing.

File: EvidenceApi/StaffSelectedDocumentTypes.json (hunk 1 above)
Comment (bella-cockrell) [reply EvidenceApi/StaffSelectedDocumentTypes.json]: Taken in!

File: EvidenceApi/StaffSelectedDocumentTypes.json (hunk 2 above)
Comment (bella-cockrell) [reply EvidenceApi/StaffSelectedDocumentTypes.json:523]: Both added!

**Pull request merged**